### PR TITLE
feature/onboard-app-to-travel-document-concept

### PR DESCRIPTION
**Pull request by gorandalum**
Fixes: https://github.com/AtB-AS/kundevendt/issues/434

### CODE REVIEW

**Comment by Nisha1306** [comment]
@gorandalum Since you had mostly reviewed it and I also implemented your comments, would be nice if you can have a quick glance at this?

**Review by gorandalum — COMMENTED** [review]
Looks good now :+1: Should merge/rebase master before merging, so the tsc action succeeds first.

**Comment by Nisha1306** [comment]
> Looks good now 👍 Should merge/rebase master before merging, so the tsc action succeeds first.

Merged with master. If all looks good, could you approve?

**Comment by Nisha1306** [comment]
> > Looks good now 👍 Should merge/rebase master before merging, so the tsc action succeeds first.
> 
> Merged with master. If all looks good, could you approve?

I realised you can not do this since you only raised this new version of PR. so I will approve and merge now 👍

File: src/screens/MobileTokenOnboarding/components/PageIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    flexDirection: 'row',
+    justifyContent: 'center',
+    marginBottom: 38.5,
+  },
+  pageDot: {
+    height: theme.spacings.medium,
+    width: theme.spacings.medium,
+    marginHorizontal: theme.spacings.medium / 2,
+    borderRadius: theme.border.radius.regular,
+    backgroundColor: '#FFFFFF',
```
[the remark this answers is not in the document]
Comment (toretefre) [reply src/screens/MobileTokenOnboarding/components/PageIndicator.tsx:34]: Or maybe @Nisha1306?

[the remark this answers is not in the document]
Comment (Nisha1306) [reply src/screens/MobileTokenOnboarding/components/PageIndicator.tsx:34]: mmm not sure. I think I copy pasted this from a draft PR. But if you want to change it to be used from the design system, you can do it :)

**Pull request merged**